src/mixins/fee-mixin.ts

```diff
@@ -66,37 +66,14 @@ export default class FeeMixin extends Vue {
   }
 
   /** Lines up promises to fetch for filing fees, falls back to empty fees.  */
-  protected async getFilingFeesFromFilingData (isFutureEffective = false): Promise<FeesIF[]> {
+  private async getFilingFeesFromFilingData (isFutureEffective = false): Promise<FeesIF[]> {
     try {
       const feePromises = this.getFilingData
-        .map(fd => this.fetchFilingFees(fd.filingTypeCode, fd.entityType, isFutureEffective))
+        .map(fd => PayServices.fetchFilingFees(fd.filingTypeCode, fd.entityType, isFutureEffective))
       return await Promise.all(feePromises)
     } catch (error) {
       console.log(error)
       return this.getFilingData.map(() => cloneDeep(EmptyFees))
     }
   }
-
-  /**
-   * Fetches filing fees.
-   * @returns a promise to return the fees for a filing
-   */
-  protected async fetchFilingFees (filingCode: FilingCodes, entityType: CorpTypeCd, isFutureEffective = false):
```

Comment:
I was not able to mock this method in the unit tests, so I moved it to a new services file. This is both consistent with other network calls, and I can mock it successfully.